### DISCUSSION TITLE
需要双击右键才能弹出右键菜单

### DISCUSSION THREAD

**Discussion by jiangcaida** [shuax/MouseInc.Settings#85] [Q&A]
需要双击右键才能弹出右键菜单

**Comment by shuax** [comment]
能稳定复现吗，我这边没有遇到这个问题。

**Comment by jiangcaida** [comment]
百分百复现，但我在另一台电脑上没有这个问题，我这边的Windows是最新的21H------------------&nbsp;原始邮件&nbsp;------------------
***@***.***&gt;
发送时间:&nbsp;2021年8月5日(星期四) 下午5:42
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [shuax/MouseInc.Settings] 需要双击右键才能弹出右键菜单 (#85)

**Comment by shuax** [comment]
说下能复现的环境吧，也许和什么软件不兼容。

**Reply by WestGe** [reply]
360安全卫士需要添加进软件检测白名单以后，才能避免右键必须双击的情况，我个人的过程是打开360极速版以后，打开右上角的【安全操作中心】，在【已信任区】里添加程序，重启软件以后就没事了。

**Comment by jiangcaida** [comment]
把360卸载了，现在好了------------------&nbsp;原始邮件&nbsp;------------------
***@***.***&gt;
发送时间:&nbsp;2021年8月5日(星期四) 下午5:50
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [shuax/MouseInc.Settings] 需要双击右键才能弹出右键菜单 (#85)
